genai/README.md

```diff
@@ -4,7 +4,8 @@ In this lab, we will go through LLM Chatbot AMP using Cloudera Machine Learning
 
 1. On login, you will be taken to Trials Home Page
 2. Click, **Start Trial** go to your Machine Learning Workspace
```

Comment:
This is actually true. For CAII you need to click on the 9dot menu on the topleft corner. Can we make a note about that?